### PR TITLE
Status mhub

### PR DESCRIPTION
**Pull request by itamargreen**
This probably has a number of bugs, but the basic functionality of sending the tournament status to https://github.com/itamargreen/tournament-status (a React web module) works.

### CODE REVIEW

**Comment by taltaub22** [comment]
@idanstark42 Please take a brief look to see if something bother you. Thanks

File: .eslintrc.yml [hunk 1]
```diff
@@ -31,10 +31,10 @@ rules:
   node/exports-style:
   - warn
   - exports
-  node/no-unsupported-features: warn
```
Comment (idanstark42) [inline .eslintrc.yml]: Why did you change the eslint?

File: module.yml [hunk 2]
```diff
@@ -6,3 +6,10 @@ type: 'node'
 script: 'index.js'
 require:
   mongodb
+config:
```
Comment (idanstark42) [inline module.yml]: How come this is defined here? It should be defined by each client

Comment (taltaub22) [reply module.yml]: That's for the mhub message.
